sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/strategies/synccommit/CommitSyncErrorHandler.java

```diff
@@ -0,0 +1,23 @@
+package org.sdase.commons.server.kafka.consumer.strategies.synccommit;
+
+import org.apache.kafka.clients.consumer.Consumer;
+
+/**
+ * Interface for handling errors that might occur when committing the offset. Be aware that the key
+ * or value can be null if the WrappedNoSerializationErrorDeserializer is used.
+ *
+ * @param <K> key class of the record
+ * @param <V> value class of the record
+ */
+@FunctionalInterface
+public interface CommitSyncErrorHandler<K, V> {
```

Comment:
nit: I'd prefer the name `SyncCommitErrorHandler` which matches `SyncCommitMLS`.